doc/themes/nilearn/layout.html

```diff
@@ -219,7 +219,7 @@
   <ul class="simple">
     <li>
       <div id="download-container">
-	<a href="{{pathto('introduction')}}#installation" class="btn btn-primary">Download</a>
+	<a href="{{pathto('introduction')}}#installation" class="btn btn-primary">Installation</a>
```

Comment:
I am not convinced by this change as "Download" is a kind of universal convention now.